openBCIBoard.js

```diff
@@ -259,6 +260,22 @@ function OpenBCIFactory() {
             boardSerial.once('open',() => {
                 var timeoutLength = this.options.simulate ? 50 : 300;
                 if(this.options.verbose) console.log('Serial port open');
+                if(dontReset) {
+                    this.streaming = false;
+                    this.curParsingMode = k.OBCIParsingNormal;
+                    var t, l;
+                    t = setTimeout(() => {
+                        this.removeListener('sample', l);
+                        resolve();
+                    },2 * 1000 / this.sampleRate());
```

Comment:
any reason this is two seconds? Could this be like 0.5 seconds or sample rate times two